data/test_data_html.json

```diff
@@ -0,0 +1,4 @@
+
+{
+  "expected_content": "Student Report\nGenerated at: 2024-02-14 22:11\n\nUsername: Cezary\nScores: [3.0, 5.0, 1.0, 3.0, 1.0, 1.0, 5.0, 4.0, 6.0]\nAverage: 3.2\nPassed: Passed\n---\n\nUsername: Dionizy\nScores: [6.0, 3.0, 6.0, 5.5, 5.0, 3.5, 4.0]\nAverage: 4.7\nPassed: Passed\n---\n\nUsername: Barnaba\nScores: [5.0, 4.0, 5.5, 4.0, 6.0, 3.5, 1.0]\nAverage: 4.1\nPassed: Passed\n---\n\n"
+}
```

Comment:
Overengineered. Just put txt and html files directly and read their content in the test